exonum/src/api/mod.rs

```diff
@@ -13,6 +13,7 @@
 // limitations under the License.
 
 //! `RESTful` API and corresponding utilities.
+// spell-checker:ignore Tful
```

Comment:
Hmmm, is this really a good idea to add all these annotations into the Rust sources?

Reply:
If we look at this specific example, `Tful` can be a typo in another file, so it's better to ignore it only locally.

Reply:
In case we decide to remove all the lints from source files, you'll be forced to add every test string (like "faedjaef") to the dictionary, which seems weird.

Reply:
Maybe Cspell has something like an external config for such per-file settings? :(

Reply:
No, there's no. Either global config, or inline lints

Reply:
That's sad. Ok, in that case, inline directives for rare special cases seem fine.

Reply:
Though, almost none of the `ignore`s in this PR seem like special cases to me. Would adding `RESTful` to the dictionary work in this case? 

We have quite a few Alexeys around here, so perhaps it shouldn't be a special case either: https://github.com/exonum/exonum/pull/453/files#diff-586e073566ba414fe35c6b99e404b3a5R15? :) 

And like almost all other cases seem similar :) 


>Tful can be a typo in another file

I would say that probability of this is negligible, and I personally would prefer a `Tful` typo in other file to this `ignore: ` thing. Like, supressing the typo creates more noise than the typo itself.

Reply:
Probably my mistake, I've tried `RESTful` before and it didn't work. 
But it works now though, so okay.

Reply:
> We have quite a few Alexeys around here, so perhaps it shouldn't be a special case either: 

What do you mean? Should I add exception to the dictionary?
Sandbox code is one special case because of its comments. I don't have enough info to fix them, but there're a lot of exceptional words in there.

Reply:
I think that the comments about "Alexey" can be easily removed.

Reply:
@DarkEld3r done